src/components/input-box.tsx

```diff
@@ -55,13 +54,19 @@ const subtext = css`
 interface InputBoxProps extends React.InputHTMLAttributes<HTMLInputElement> {
   label?: string;
   note?: string;
+  onChangeInput?: (value: string) => void;
 }
 
-export default function InputBox({ label, note, ...rest }: InputBoxProps) {
+export default function InputBox({ label, note, onChangeInput, ...rest }: InputBoxProps) {
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    // console.log(e.target.value);
+    onChangeInput && onChangeInput(e.target.value);
```

Comment:
```suggestion
    onChangeInput?.(e.target.value);
```

요렇게 함수 호출할 때에도 옵셔널 체이닝을 사용할 수 있습니다!

Reply:
와 요건 몰랐네요 0_0 또 하나 배워갑니다 😆